product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/utils/XMLComparatorInterop.java

```diff
@@ -0,0 +1,212 @@
+package org.wso2.carbon.esb.scenario.test.common.utils;
+
+import org.apache.axiom.om.OMAttribute;
+import org.apache.axiom.om.OMElement;
+import org.apache.axiom.om.OMNamespace;
+import org.apache.axiom.om.OMNode;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.util.Iterator;
+import java.util.Vector;
+
+/**
+ * Util class for comparing two OMElements
+ */
+public class XMLComparatorInterop {
+
+    private static final Log log = LogFactory.getLog(XMLComparatorInterop.class);
```

Comment:
Let's have the correct naming convention for static final constants

Reply:
I prefer "log", opposed to LOG as it is easy to write. :). I will change as it goes with Java conventions.